pwiz_tools/Skyline/Model/AbstractDdaSearchEngine.cs

```diff
@@ -36,13 +36,13 @@ public abstract class AbstractDdaSearchEngine : IDisposable
         public abstract string[] FragmentIons { get; }
         public abstract string EngineName { get; }
         public abstract Bitmap SearchEngineLogo { get; }
-        protected MsDataFileUri[] SpectrumFileNames { get; set; }
+        public MsDataFileUri[] SpectrumFileNames { get; protected set; }
         protected string[] FastaFileNames { get; set; }
 
         /// <summary>
         /// A string or numeric variable with a default value and, for numeric types, min and max values.
         /// </summary>
-        public class Setting
+        public class Setting : IAuditLogObject
```

Comment:
I am not an expert on audit log objects, but I was under the impression that if a class had a self-describing ToString method, it would not need to implement IAuditLogObject.